mutagen-core/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(proc_macro_span)]
+#![feature(box_syntax)]
+#![feature(vec_remove_item)]
```

Comment:
Also this?

Reply:
This is used during configuration of the transformers. I could replace its use with the 2-line implementation of `remove_item`. We are on nightly already and I think those features are quite stable.

The features `proc_macro_span` and `specialization` are necessary for the current approach to be implementable. I thought that the extra features do not add extra risk to the project.

Reply:
Agreed.